primitives/src/sentry.rs

```diff
@@ -22,9 +22,15 @@ pub use event::{Event, EventType, CLICK, IMPRESSION};
 
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
 #[serde(rename_all = "camelCase")]
-/// Channel Accounting response.
+/// Channel Accounting response
 ///
-/// A collection of all `Accounting`s for a specific [`Channel`](crate::Channel).
+/// A collection of all `Accounting`s for a specific [`Channel`](`crate::Channel`)
```

Comment:
@simzzz for next time be aware:

```suggestion
/// A collection of all `Accounting`s for a specific [`Channel`](crate::Channel)
```